cmd/utils/flags.go

```diff
@@ -781,8 +793,11 @@ func setHTTP(ctx *cli.Context, cfg *node.Config) {
 		}
 	}
 
-	if ctx.GlobalIsSet(RPCPortFlag.Name) {
+	switch {
+	case ctx.GlobalIsSet(RPCPortFlag.Name):
 		cfg.HTTPPort = ctx.GlobalInt(RPCPortFlag.Name)
+	case ctx.GlobalBool(MixFlag.Name):
+		cfg.HTTPPort = 8645
```

Comment:
RPC / HTTP port should not be hardcoded.  This is user-configurable, the user can choose any port arbitrarily, and it will continue to work properly with MIX Blockchain